configs/IBM-POWER9-V100-Infiniband/system_definition.yaml

```
@@ -0,0 +1,28 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: IBM-POWER9-V100-Infiniband
+  manufacturer:
+    vendor: IBM
+    name: AC922
+  processor:
+    vendor: IBM
+    name: POWER9
+    ISA: POWER-3.0B
```

```suggestion
    ISA: power9
    ISA_family: ppc64le
```

This keeps our descriptions in line with what you get from https://github.com/archspec/archspec.git

---

Yes, thanks.  Is there a full list of the strings I can look up for capitalization?

But oh no, you are breaking my non-existent schema lol.  ISA then ISA_family in one file, switched order in the next file.  This will get messy quick if we don't write that schema soon.